src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-card flat id="extend-time-limit" class="pt-2 mr-8">
+
+    <!-- Extension Time Section -->
+    <section
+      id="extension-time-section"
+      class="section-container"
+      :class="{ 'invalid-section': invalidSectionExpiry }"
+    >
+      <v-row no-gutters>
+        <v-col cols="12" sm="3">
+          <strong :class="{ 'error-text': invalidSectionExpiry }">Extension Time</strong>
+        </v-col>
+        <v-col cols="12" sm="9">
+          <!-- Limited Restoration Radio Panel -->
+          <LimitedRestorationPanel
+            :currentDate="getCurrentDate"
+            :expiryDate="expiry"
+            :key="expiry"
+            :maxNumberOfMonths=36
+            @expiry="setRestorationExpiry(addMonthsToDate(previousNumberOfMonths, $event))"
```

Comment:
New expiry is going to be:
The expiry of the previously filed limited restoration + the new extension period.

Reply:
3 years now? Does Create UI have to be updated for the initial limited resto application? (The shared component defaults it to 24.)

Reply:
> New expiry is going to be: The expiry of the previously filed limited restoration + the new extension period.

Did you confirm this? It doesn't sound right to me.

Reply:
> 3 years now? Does Create UI have to be updated for the initial limited resto application? (The shared component defaults it to 24.)

According to UXPin:
![uxpin extension](https://user-images.githubusercontent.com/122301442/228028653-a996b84a-8cda-4a9c-8e4f-3333a7e24c80.PNG)
For Create UI: It's 24 months.
For Edit UI: it's 36 months.

Reply:
> > New expiry is going to be: The expiry of the previously filed limited restoration + the new extension period.
> 
> Did you confirm this? It doesn't sound right to me.

I spoke with Ethan about it. He told me it's business logic so I should ask Mihai.
I asked Mihai and he confirmed that this is the way they want to have it!

Reply:
OK. Thanks for checking. I'm OK with this.